docs/resources/server/advancements.md

```diff
@@ -63,8 +63,8 @@ public ExampleTriggerInstance(EntityPredicate.Composite player, ItemPredicate it
 Typically, trigger instances have a static constructor which allow these instances to be easily created for data generation. These static factory methods can also be statically imported instead of the class itself.
 
 ```java
-public static ExampleTriggerInstance instance(EntityPredicate.Builder playerBuilder, ItemPredicate.Builder itemBuilder) {
-  return new ExampleTriggerInstance(EntityPredicate.Composite.wrap(playerBuilder.build()), itemBuilder.build());
+public static ExampleTriggerInstance instance(ContextAwarePredicate player, ItemPredicate item) {
+    return new ExampleTriggerInstance(player, item);
```

Comment:
I just realized, the indentation in the rest of the examples use 2 space. Go back, go back!
```suggestion
  return new ExampleTriggerInstance(player, item);
```